src/utils.py

```diff
@@ -11,11 +11,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def ReadFile(file_path: str) -> None:
-    with open(file_path) as file:
-        for line in file:
-            output = line
+import linecache
+from random import randrange
 
+def ReadRandomLine(file_path: str) -> None:
+    with open(file_path) as file:
+        for count, line in enumerate(file):
```

Comment:
does 'line' do anything here? If not, delete

Reply:
This was just so I could extract the count from the tuple returned by enumerate. I've reworked to avoid the obsolete variable by using count[0]